.github/workflows/ci.yml

```diff
@@ -119,7 +119,7 @@ jobs:
 
       - name: Export deployments
         run: |
-          for NETWORK in bsctestnet bscmainnet ethereum sepolia opbnbtestnet opbnbmainnet; do
+          for NETWORK in bsctestnet bscmainnet ethereum sepolia opbnbtestnet opbnbmainnet arbitrumSepolia arbitrumOne; do
```

Comment:
I would use lower case, as we are doing for the rest of the networks

Reply:
Done https://github.com/VenusProtocol/venus-protocol/pull/456/commits/070012779c9bf4a5a306e9cdc4133c986e450b95